e2e/public-web/test_contact_form.py

```
@@ -0,0 +1,251 @@
+            print("  [PASS] Back button visible")
+            print("\n[NOTE] Form submission not tested to avoid creating test data in production")
+
+            return True
+
+        except AssertionError as e:
+            print(f"\n[ASSERTION ERROR] {e}")
+            take_screenshot(page, "public_contact_error_assertion", "Assertion error")
+            traceback.print_exc()
+            return False
+
+        except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)